src/components/host/host.js

```diff
@@ -26,10 +25,10 @@ function Host() {
   };
   return (
     <div>
-      <h1 className="title">Host a Period Dignity Box</h1>
       <div className="container-circle top-section row">
+        <h1 className="host-title row">Host a Period Dignity Box</h1>
         <div className="top-text-left column">
-          <Person className="person" />
+          <Person className="person-host" alt=""/>
```

Comment:
Do we have alternative text to place yet?

Reply:
@msajara advised to leave it blank as they are decorative.

Reply:
👍